src/test/java/uk/gov/hmcts/ccd/v2/external/controller/TestingSupportControllerTestIT.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.hmcts.ccd.v2.external.controller;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.gov.hmcts.ccd.MockUtils;
+import uk.gov.hmcts.ccd.WireMockBaseTest;
+import uk.gov.hmcts.ccd.data.caseaccess.CaseLinkEntity;
+import uk.gov.hmcts.ccd.data.caseaccess.CaseLinkRepository;
+import uk.gov.hmcts.ccd.domain.model.casedeletion.CaseLink;
+import uk.gov.hmcts.ccd.domain.model.casedeletion.CaseLinksResource;
+
+import javax.inject.Inject;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(MockitoExtension.class)
```

Comment:
`@ExtendWith(MockitoExtention.class)` looks redundant here

Reply:
Done in [fe6c52f](https://github.com/hmcts/ccd-data-store-api/pull/1719/commits/fe6c52f4bc3d7709a1a740daa2b74ad122b5a4a1)